variables.tf

```diff
@@ -120,3 +120,8 @@ variable "gitlab_group_id" {
   description = "Gitlab group id"
   type        = string
 }
+
+variable "tag_list" {
+  description = "Tag for the agent runner"
+  type        = string
```

Comment:
Please add
default = "Platform Engineering"

Reply:
Just want to confirmg that you mean `platform-engineering`, right?